### PR TITLE
Statistics for boosters (EXPOSUREAPP-10765)

### PR DESCRIPTION
**Pull request by SamuraiKek**
Added a new statistics card that shows the number of people with boosters.

To test it, switch to `MOCK_CLOUD` and the card should be second to last. Don't mind the values, they're inflated from the mock server.

Also, make sure it works in night mode and that all the people on the cards are now wearing a mask in both light/dark mode.

### CODE REVIEW

**Review by AlexanderAlferov — CHANGES_REQUESTED** [review]
I think new text is missing in the Info Screen ℹ️ Kindly check the Figma 😄

**Comment by SamuraiKek** [comment]
> I think new text is missing in the Info Screen ℹ️ Kindly check the Figma 😄

Thanks, I always forget about it 😄

**Review by AlexanderAlferov — DISMISSED** [review]
LGTM 🚀

**Review by CV113 — DISMISSED** [review]
UA approved

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/StatsItem.kt [hunk 1]
```diff
@@ -238,6 +239,28 @@ data class PersonsVaccinatedCompletelyStats(
...
+    val boosterDoses: KeyFigure
+        get() = keyFigures.single { it.rank == KeyFigure.Rank.PRIMARY }
+
+    val total: KeyFigure
+        get() = keyFigures.single { it.rank == KeyFigure.Rank.TERTIARY }
+
+    override fun requireValidity() {
+        require(keyFigures.size == 2)
+        requireNotNull(keyFigures.singleOrNull { it.rank == KeyFigure.Rank.PRIMARY }) {
+            Timber.w("PersonsVaccinatesWithBoostersStats is missing primary value")
```
Comment (jurajkusnier) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/StatsItem.kt:256]: PersonsVaccinatesWithBoostersStats > PersonsVaccinatedWithBoosterStats
typo?

**Review by jurajkusnier — COMMENTED** [review]
Could you double-check that Android texts are updated? There is a mismatch between iOS and Android design
out

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/StatsItem.kt (hunk 1 above)
Comment (SamuraiKek) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/StatsItem.kt:256]: Yes, good catch!

**Comment by SamuraiKek** [comment]
> Could you double-check that Android texts are updated? There is a mismatch between iOS and Android design 

Turns out the iOS one is correct. I will update it.

**Review by CV113 — APPROVED** [review]
UA approved

**Review by jurajkusnier — APPROVED** [review]

**Review by AlexanderAlferov — APPROVED** [review]

**Pull request merged**